details.html

```diff
@@ -769,7 +769,7 @@ <h2 class="content-title">checklist (50%)</h2>
                     <span class="material-icons-round">add</span>
                     Add Checklist Item
                   </button>
```

Comment:
Read it until you understand it https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul